.github/workflows/tests.yml

```diff
@@ -201,7 +189,11 @@ jobs:
           - runner: '11'
           - runner: '10.15'
           - runner: ${{needs.setup_tests.outputs.linux-runner}}
-            container: ${{fromJson(needs.setup_tests.outputs.container)}}
+            container:
+              image: ghcr.io/homebrew/ubuntu16.04:master
+              env:
+                GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED: 1
```

Comment:
This does not seem correct. The option before allowed env passthrough so that if it is set elsewhere it would be passed into the container.

It is not correct for it to _always_ be set since we use GitHub hosted runners too.

Reply:
Right, fixed.